### PR TITLE
Doc patch and proj rename

### PR DESCRIPTION
**Pull request by brunoborges**

### CODE REVIEW

**Comment by brunoborges** [comment]
@johnoliver there it goes. Still not perfect, but advancing...

File: java/README.md [hunk 1]
```diff
@@ -10,34 +10,59 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
...
-To build the semantic kernel, you will need to have:
+To build the project, you will need to have:
 
-- **Required**:
-    - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
-    - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+- **Required**: 
+  - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
+  - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+  - [Apache Maven](https://maven.apache.org/) 3.9.1 or newer
```
Comment (johnoliver) [inline java/README.md]: we have mvnw so installing maven is not required

File: java/README.md [hunk 2]
```diff
@@ -10,34 +10,59 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
...
+  - Azure OpenAI SDK for Java _(see below for instructions)_
 
-### Build the semantic kernel
+## Building the Semantic Kernel
+
+The Semantic Kernel is built using [Apache Maven](https://maven.apache.org/). To build it you will first need to have the Azure OpenAI SDK for Java installed in your local Maven repository. At this moment, this SDK must be built locally from the source.
+
+### Build the Azure OpenAI SDK locally
```
Comment (johnoliver) [inline java/README.md]: You dont need to install the SDK manually, maven will download it

File: java/README.md [hunk 3]
```diff
@@ -10,34 +10,59 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
...
         git clone git@github.com:microsoft/semantic-kernel.git
 
-2. Build the semantic kernel
+2. Build the Semantic Kernel
 
         cd semantic-kernel/java
         mvn install
 
-## Using the semantic kernel
+    - _Note: disable the integration tests if the build of fails for lack of an OpenAI key_
```
Comment (johnoliver) [inline java/README.md]: Tests that require a key will not run by default

Comment (brunoborges) [reply java/README.md]: Fixed in ad5ebd4

File: java/README.md (hunk 2 above)
Comment (brunoborges) [reply java/README.md]: Fixed in ad5ebd4

File: java/README.md (hunk 1 above)
Comment (brunoborges) [reply java/README.md]: Fixed in ad5ebd4

File: FEATURE_MATRIX.md [hunk 4]
```diff
@@ -1,61 +1,61 @@
...
-| TextMemorySkill                   | ✅ | ✅ | |
-| ConversationSummarySkill          | ✅ | ❌ | |
-| FileIOSkill                       | ✅ | ✅ | |
-| HttpSkill                         | ✅ | ✅ | |
-| MathSkill                         | ✅ | ❌ | |
-| TextSkill                         | ✅ | ✅ | |
-| TimeSkill                         | ✅ | ✅ | |
+| | C# | Python | Java | Notes |
+|---|---|---|---|---|
+| TextMemorySkill                   | ✅ | ✅ | ✅ | |
```
Comment (johnoliver) [inline FEATURE_MATRIX.md]: will need to ask David on this but I believe that may still be considered wip

File: FEATURE_MATRIX.md [hunk 5]
```diff
@@ -1,61 +1,61 @@
...
+| ConversationSummarySkill          | ✅ | ❌ | ❌ | |
+| FileIOSkill                       | ✅ | ✅ | ❌ | |
+| HttpSkill                         | ✅ | ✅ | ❌ | |
+| MathSkill                         | ✅ | ❌ | ❌ | |
+| TextSkill                         | ✅ | ✅ | ✅ | |
```
Comment (johnoliver) [inline FEATURE_MATRIX.md]: This is wip

File: java/README.md [hunk 6]
```diff
@@ -10,34 +10,41 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
...
-To build the semantic kernel, you will need to have:
+To build the project, you will need to have:
 
-- **Required**:
-    - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
-    - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+- **Required**: 
+  - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
+  - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+  - Azure OpenAI SDK for Java _(see below for instructions)_
```
Comment (johnoliver) [inline java/README.md]: this is installed by default, not required

File: FEATURE_MATRIX.md (hunk 5 above)
Comment (brunoborges) [reply FEATURE_MATRIX.md:35]: Which one? TextSkill?

File: FEATURE_MATRIX.md (hunk 4 above)
Comment (brunoborges) [reply FEATURE_MATRIX.md:30]: TextMemorySkill?

File: java/README.md (hunk 6 above)
Comment (brunoborges) [reply java/README.md]: Missed that line.

Comment (brunoborges) [reply java/README.md]: Fixed in 37c9e30.

File: FEATURE_MATRIX.md (hunk 5 above)
Comment (brunoborges) [reply FEATURE_MATRIX.md]: Fixed in 37c9e30.

File: FEATURE_MATRIX.md (hunk 4 above)
Comment (brunoborges) [reply FEATURE_MATRIX.md]: Fixed in 37c9e30.

**Review by johnoliver — APPROVED** [review]

**Pull request merged**